### PR TITLE
Additional information on system chains and collectives

### PR DESCRIPTION
**Pull request by alfarok**
This PR addresses #4094, originating from #4261 with the new changes and re-org from #4262

- [x] Cleanup pass over legacy learn-common-good-chains doc
- [x] Add some additional info on Collectives
- [x] Elaborate on how parachains impact bridging

### CODE REVIEW

**Comment by alfarok** [comment]
Replacing w/ #4261 after the re-org and updates from #4262.

File: docs/learn/learn-system-chains.md [hunk 1]
```diff
@@ -104,10 +123,19 @@ Polkadot network.
...
+within other networks. This is achieved with the assistance from a [bridge hub](#bridge-hubs).
+
+### Bridge Hubs
 
-### Bridge Hub
+Before Polkadot and Kusama supported their first parachains, the only way to design a bridge was to
+put the logic onto the Relay Chain itself. Since both networks now support parachains, it makes
+sense to have a parachain on each network dedicated to bridges. This is because of the execution
+isolation provided by parachains. The activity on a parachain does not affect the Relay Chain or
+other parachains, so a bridge hub can support bridges to many other consensus systems.
```
Comment (profradha) [inline docs/learn/learn-system-chains.md]: Removing the ambiguous statement. The activity on parachain does affect the relaychain and other parachains (through XCM). Probably the sentence can benefit from rephrasing or be removed completely. 

```suggestion
isolation provided by parachains.
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Looks good. Minor changes requested.

**Comment by alfarok** [comment]
Thanks @DrW3RK, suggestion applied!

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**